extensions/http/src/jolie/net/HttpProtocol.java

```diff
@@ -202,6 +203,7 @@
 		private static final String CONTENT_DISPOSITION = "contentDisposition";
 		private static final String DROP_URI_PATH = "dropURIPath";
 		private static final String CACHE_CONTROL = "cacheControl";
+		private static final String STRING_RAW_CONTENT = "stringRawContent";
```

Comment:
Sounds good then, go ahead. @jolie/developers please pitch in if you have better ideas.